.github/workflows/CI.yml

```diff
@@ -66,12 +62,8 @@ jobs:
       env:
         ZEEBE_ADDRESS: test.example.com:443
       run: |
-        npm run test:secure | tee output.txt
-        results=$(grep "\"gatewayVersion\": \"${ZEEBE_VERSION}\"" -c output.txt)
-        if [[ "$results" != "2" ]]; then
-          echo "missing <gatewayVersion>"
-          exit 1
-        fi
+        set -o pipefail
+        npm run test:secure | tee output.txt ; echo $?
```

Comment:
I assume that `npm run test:**` execute with a non 0 exit code? Or how do we detect errors now?

Reply:
Before the change, `tee` would always return 0 exit code. So even if `npm run test:secure` failed, the job would succeed on GH Actions. With `set -o pipefail`, the pipe keeps non-zero exit code of the first command.

Reply:
> pipe keeps non-zero exit code of the first command.

This I understand. The previous test was not meant to propagate the exit code, but test the expected system behavior (gateway version reported). :slightly_smiling_face: 

What I'm saying is that we now rely on propagation of exit code, which I'm OK with. I just want to make you aware that we change the test strategy.

Reply:
Let me check if we can make it work without change of the test strategy.

Reply:
No magic-number solution: https://github.com/camunda/zeebe-connection-test/pull/12/commits/806cf3a0aca9d79393e58b21b34ca32657b3454c

Reply:
So now build fails because updated `zbctl` prints double spaces between key and value 🙄

Reply:
```json
> zeebe-tls-connection-test@0.1.0 test:zbctl
> zbctl status -o json

{
  "brokers":  [
    {
      "nodeId":  0,
      "host":  "0.0.0.0",
      "port":  26501,
      "partitions":  [
        {
          "partitionId":  1,
          "role":  "LEADER",
          "health":  "HEALTHY"
        }
      ],
      "version":  "8.5.1"
    }
  ],
  "clusterSize":  1,
  "partitionsCount":  1,
  "replicationFactor":  1,
  "gatewayVersion":  "8.5.1"
}
```

Reply:
Hahaha. Well, if we want to change our test strategy... :wink: 

Of course there is other ways to solve this, i.e. using [jq](https://jqlang.github.io/jq/) to parse the value returned.

Reply:
Fixed via https://github.com/camunda/zeebe-connection-test/pull/12/commits/aad4341b631e4e5e4c75d1b5af610a120ba2a51a